packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -290,7 +290,7 @@ export class RunningSummarizer implements IDisposable {
      */
     public async lockedRefreshSummaryAckAction<T>(action: () => Promise<T>) {
         assert(this.refreshSummaryAckLock === undefined,
-            "Refresh Summary Ack - Caller is responsible for checking lock");
+            0x347 /* Refresh Summary Ack - Caller is responsible for checking lock */);
```

Comment:
can you make sure this matches main. we shouldn't be generating new hex codes in release branches